test/acceptance/specs/BaseSpec.scala

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package acceptance.specs
+
+import org.scalatest._
+import org.scalatest.concurrent.Eventually
+import org.scalatest.featurespec.AnyFeatureSpec
+import org.scalatest.matchers.should.Matchers
+import org.scalatestplus.selenium.WebBrowser
+import uk.gov.hmrc.webdriver.SingletonDriver
+import acceptance.driver.BrowserDriver
+
+import scala.util.Try
+
+trait BaseSpec
+    extends AnyFeatureSpec
+    with GivenWhenThen
+    with BeforeAndAfterAll
+    with Matchers
+    with WebBrowser
+    with BrowserDriver
```

Comment:
For this to work locally and in Jenkins you would need to mix in a TestAcceptanceServer like [here](https://github.com/hmrc/tracking-consent-frontend/blob/cfc9a17b709f7ff8c63a0924b160e5243e60ec45/test/acceptance/specs/BaseAcceptanceSpec.scala#L34) – ideally we could do with this living in a library somewhere.